### ISSUE TITLE
Circular/pie timer progressbar

### ISSUE THREAD

**Issue by atomeon** [kevinsqi/react-circular-progressbar#154]
### It would be nice to have any way to make timer/countdown just by passing some props. 

i have a task to make 10 sec circular progress timer, and what i made according to examples is showing progress only after number of percentage has increased. And i need to slowly increase timer animation before number has increased. What i searching for is more closer to this example https://codepen.io/zmuci/pen/rJooKr
